dags/openshift_nightlies/config/install/rosa/preflight.json

```diff
@@ -0,0 +1,16 @@
+{
+    "aws_profile": "",
+    "aws_access_key_id": "",
+    "aws_secret_access_key": "",
+    "aws_authentication_method": "sts",
+    "rosa_environment": "staging",
+    "rosa_cli_version": "master",
+    "ocm_environment": "stage",
+    "rosa_channel_group": "stable",
+    "rosa_ocp_version": "4.10.12",
+    "openshift_worker_count": 27,
```

Comment:
Why are we testing this w/ 27 nodes for the install? I would think the minimum 3 would do given we're really just looking at the preflight times.

Reply:
Totally right